domains/arabiancoconut.is-cool.dev.json

```diff
@@ -0,0 +1,20 @@
+{
+    "schema": "../schemas/domain.schema.json",
+    "description": "The domain for project use",
```

Comment:
A description of the site would be needed here.

Reply:
Not required.